### DISCUSSION TITLE
v5.1.1

### DISCUSSION THREAD

**Discussion by zkoppert** [super-linter/super-linter#4440] [Releases]
# Changelog
## 🧰 Maintenance

- Update README.md @kerwin612 (#4393)

See details of [all code changes](https://github.com/github/super-linter/compare/v5.1.0...v5.1.1) since previous release

<hr /><em>This discussion was created from the release <a href='https://github.com/super-linter/super-linter/releases/tag/v5.1.1'>v5.1.1</a>.</em>

**Comment by grahame-student** [comment]
Is it intentional that the slim metadata file still references the slim v5.0.0 docker image and not the v5.1.1?

See #4494

**Comment by zvailanu98** [comment]
Maintainer

**Comment by zvailanu98** [comment]
. If you need to, you can also  or learn more about diff comparisons.
 
...
 
There isn’t anything to compare.
